jboss-eap-xp1-openjdk11-openshift.json

```diff
@@ -0,0 +1,123 @@
+                    "openshift.io/provider-display-name": "Red Hat",
+                    "version": "1.0"
+                }
+            },
+            "labels": {
+                "xpaas": "1.0"
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "latest",
+                        "annotations": {
+                            "description": "The latest released build of JBoss EAP XP",
+                            "iconClass": "icon-eap",
+                            "tags": "builder,eap-xp,javaee,java,jboss,hidden",
+                            "supports": "eap-xp:1,javaee:8,java:11",
+                            "version": "latest",
+                            "openshift.io/display-name": "JBoss EAP XP with OpenJDK 11"
+                        },
+                        "referencePolicy": {
+                            "type": "Local"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "registry.redhat.io/jboss-eap-7/eap-xp1-openjdk11-openshift-rhel8:latest"
+                        }
+                    },
+                    {
+                        "name": "1.0",
+                        "annotations": {
+                            "description": "JBoss EAP XP Builder Image",
+                            "iconClass": "icon-eap",
+                            "tags": "builder,eap-xp,javaee,java,jboss,hidden",
+                            "supports": "eap-xp:1,javaee:8,java:11",
+                            "version": "1.0",
+                            "openshift.io/display-name": "JBoss EAP XP Builder Image with OpenJDK 11"
+                        },
+                        "referencePolicy": {
+                            "type": "Local"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "registry.redhat.io/jboss-eap-7/eap-xp1-openjdk11-openshift-rhel8:latest"
+                        }
+                    }
+                ]
+            }
+        },
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "jboss-eap-xp1-openjdk11-runtime-openshift",
+                "annotations": {
+                    "openshift.io/display-name": "JBoss EAP XP Runtime Image with OpenJDK 11",
+                    "openshift.io/provider-display-name": "Red Hat",
+                    "version": "1.0"
+                }
+            },
+            "labels": {
+                "xpaas": "1.0"
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "latest",
+                        "annotations": {
+                            "description": "The latest released JBoss EAP XP runtime image.",
```

Comment:
should be "Runtime image for JBoss EAP expansion pack 1.0 to run Microservices applications on RHEL8 with OpenJDK11"